nginx/proximis.conf

```diff
@@ -0,0 +1 @@
+client_max_body_size 10M;
```

Comment:
To be confirmed, but it should be

```
server {
    client_max_body_size 10M;
}
```

Reply:
It's included in http section for all server